server/notebooks/names.py

```diff
@@ -0,0 +1,26 @@
+import random
+
+ELEMENTS = [
+    ("oxygen", "oxide"),
+    ("hydrogen", "hydride"),
+    ("argon", "argide"),
+    ("phosphorous", "phosphide"),
```

Comment:
We should add `("carbon", "carbide")` so we can get some tetracarbon notebooks. ;)

Reply:
Would be a good idea, but I think we'll go with @bcolloran's plan of using a random set of iodine compounds. :)